### DISCUSSION TITLE
Alternatives to simpletask now that it is archived

### DISCUSSION THREAD

**Discussion by mshuflin** [ransome1/sleek#703] [General]
Simpletask repo is archived, and while it still works for now, I am looking for alternatives to serve as my companion to Sleek on mobile. Here are some of options I have looked at, additional suggestions are appreciated.

- [simpletask](https://github.com/mpcjanssen/simpletask-android) (unmaintained, archived)
- [todo.txt](https://play.google.com/store/apps/details?id=net.c306.ttsuper) (not open source) 
- [markor](https://github.com/gsantner/markor) (more text-based than gui, not only for tasks)
- [ntodotxt](https://github.com/tmaegel/ntodotxt) (modern ui, but missing several features, under active development)

**Comment by azzamsa** [comment]
Yeah, love sleek so much but hard to manage the task in mobile.

**Comment by fredofredofred** [comment]
Update - I wanted to check out Todo.txt but it has disappeared from the Google Play store, not sure what's going on with that.

I currently use Markor, it doesn't have good filtering options which is an issue because I have lots of tasks.

~~I might give ntodotxt a try and see how it goes, I primarily use Sleek on PC so as long as the web app has basic features I can live with that. Hopefully this can become a companion to Sleek.~~

Update:
ntodo.txt doesn't easily work with my cloud service so I'll continue with Markor (I usually open the cloud app and then select 'open with' or 'edit with' but ntodo.txt isn't an option for opening/editting files).
